packages/components/src/template-registry.ts

```diff
@@ -101,6 +103,15 @@ export default interface HdsComponentsRegistry {
   'hds/reveal/toggle/button': typeof HdsRevealToggleButtonComponent;
   HdsRevealToggleButtonComponent: typeof HdsRevealToggleButtonComponent;
 
+  // Stepper
+  'Hds::Stepper::Step::Indicator': typeof HdsStepperIndicatorStepComponent;
+  'hds/stepper/step/indicator': typeof HdsStepperIndicatorStepComponent;
+  HdsStepperStepIndicator: typeof HdsStepperIndicatorStepComponent;
```

Comment:
I got kinda dizzy trying to ensure I got `Stepper` `Step` & `Indicator` In the right order here. I _think_ it's correct but a second set of eyes would be helpful. Ditto the section below as well.